actors/common/src/main/java/org/sunbird/learner/util/UserUtility.java

```diff
@@ -36,7 +37,9 @@
     String addressKey = PropertiesCache.getInstance().getProperty("addresskey.encryption");
     addressKeyToEncrypt = new ArrayList<>(Arrays.asList(addressKey.split(",")));
     String userKeyDecrypt = PropertiesCache.getInstance().getProperty("userkey.decryption");
+    String userKeyToMasked=PropertiesCache.getInstance().getProperty("userkey.masked");
```

Comment:
better to have a init method that will do all this setting and will call that method from static block